decimal64math.go

```diff
@@ -5,8 +5,128 @@ func (d Decimal64) Abs() Decimal64 {
 	return Decimal64{^neg64 & uint64(d.bits)}
 }
 
-// Add computes d + e
+// Add computes d + e with default rounding set to half_up
 func (d Decimal64) Add(e Decimal64) Decimal64 {
+	return defaultContext.Add(d, e)
+}
+
+// FMA computes d*e + f with default rounding set to half_up
+func (d Decimal64) FMA(e, f Decimal64) Decimal64 {
+	return defaultContext.FMA(d, e, f)
+}
+
+// Mul computes d * e with default rounding set to half_even
+func (d Decimal64) Mul(e Decimal64) Decimal64 {
+	return Context64{roundHalfEven}.Mul(d, e)
+}
+
+// Sub returns d - e.
+func (d Decimal64) Sub(e Decimal64) Decimal64 {
+	return d.Add(e.Neg())
+}
+
+// Cmp returns:
+//
+//   -2 if d or e is NaN
+//   -1 if d <  e
+//    0 if d == e (incl. -0 == 0, -Inf == -Inf, and +Inf == +Inf)
+//   +1 if d >  e
+//
+func (d Decimal64) Cmp(e Decimal64) int {
+	flavor1, _, _, significand1 := d.parts()
+	flavor2, _, _, significand2 := e.parts()
+	if flavor1 == flSNaN || flavor2 == flSNaN {
+		return -2
+	}
+	if flavor1 == flQNaN || flavor2 == flQNaN {
+		return -2
+	}
+	if significand1 == 0 && significand2 == 0 {
+		return 0
+	}
+	if d == e {
+		return 0
+	}
+	d = d.Sub(e)
+	return 1 - 2*int(d.bits>>63)
+}
+
+// Neg computes -d.
+func (d Decimal64) Neg() Decimal64 {
+	return Decimal64{neg64 ^ uint64(d.bits)}
+}
+
+// Quo computes d / e.
+func (d Decimal64) Quo(e Decimal64) Decimal64 {
```

Comment:
No rounding implications?

Reply:
Nope, none at all, 
Haven't done any of the tests for these yet